catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/json/MetacardTypeSerializer.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.source.solr.json;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.MetacardType;
+import java.io.IOException;
+
+public class MetacardTypeSerializer extends StdSerializer<MetacardType> {
+
+  public MetacardTypeSerializer() {
+    super(MetacardType.class);
+  }
+
+  @Override
+  public void serialize(MetacardType value, JsonGenerator gen, SerializerProvider provider)
+      throws IOException {
+    gen.writeStartObject();
+    gen.writeStringField("name", value.getName());
```

Comment:
✏️ I should comment about making these names constants and shared between the two ... but I won't 😉

Reply:
I realized that after I pushed up the last set of changes.  I don't think a constant class is be much better.  I could make them package private in one and reuse in the other.  Is there a better alternative I am forgetting?

Reply:
I didn't say anything :-)
How about defining them as inner classes of the same container class?

Reply:
I moved the `MetacardType` `Serializer` and `Deserializer` into a `ObjectMapper` factory.  That cleaned up a few issues.